app.py

```diff
@@ -284,12 +377,16 @@ def annotate_post():
 """
 @app.route('/annotate/', methods=['GET'])
 def annotate_get():
+    if not 'model' in request.args:
+        abort(400)
```

Comment:
I would add some backwards compatibility for this: if there's only one model trained, and there's no `model` specified in the request, then just use the one and only model available.

Reply:
Done - I have added the backwards compatibility as you have described. :heavy_check_mark: